pages/blog/swr-v2.en-US.mdx

```diff
@@ -0,0 +1,262 @@
+---
+image:
+description: 'We are excited to announce SWR v2.0 release, with various improvements of DX and new features like preloading, stale, optimistic update, etc.'
+date:
+---
+
+import Callout from 'nextra-theme-docs/callout'
+import Bleed from 'nextra-theme-docs/bleed'
+
+import Authors, { Author } from 'components/authors'
+import Video from 'components/video'
+
+# Announcing SWR 2.0
+
+<Authors date="">
+</Authors>
+
+We are excited to introduce SWR v2.0, including features to improve the use experience of your application.
+
+## What’s New
+
+### Better React 18 Support
```

Comment:
We can move this section to the bottom, since there's no new API change, basically improvement. We want to show users the explict changes first which can make it a bit more attractive